dir.props

```diff
@@ -452,6 +453,12 @@
         <NuGetTargetMoniker>.NETCoreApp,Version=v1.0</NuGetTargetMoniker>
       </PropertyGroup>
     </When>
+    <When Condition="'$(TargetGroup)'=='netcoreapp1.1'">
```

Comment:
In that case, yeah we should. And another thing we'd want to do is modify some of the targets in buildtools to assign netcoreapp1.1 as the default TestTFM when under netstandard1.7. Right now there isn't really a relationship between the two.